src/styles/index.scss

```diff
@@ -0,0 +1,218 @@
+@import './global.scss';
+
+div.homeContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-top: 70px;
+  header {
+    margin: 41px 37px 0 37px;
+    >h1 {
+      color: $ashgray;
+      text-align: center;
+      font-family: Inter;
+      font-size: 34px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 37.579px;
+      margin: 0;
+      span {
+        font-family: 'ClashDisplay-Medium';
+      }
+      span:first-of-type {
+        color: $amber;
+      }
+      span:nth-of-type(2) {
+        color: $pumpkin;
+        font-family: 'ClashDisplay-Semibold';
+      }
+      span:nth-of-type(3) {
+        color: $lilac;
+      }
+      span:nth-of-type(4) {
+        font-family: Inter;
+        font-weight: 700;
+        border-radius: 50px;
+        position: relative;
+        &::after {
+          content: "";
+          display: block;
+          border: 1px solid $pumpkin;
+          border-radius: 50px;
+          position: absolute;
+          top: 10px;
+          left: -7px;
+          right: 0;
+          bottom: 0;
+          width: 187px;
+          height: 25px;
```

Comment:
Never use width and height for borders, globally you should avoid using "hard" coded sizes, it should always adapt to content.